dev/Materials/Reveal/InteractionTests/Reveal_InteractionTests/RevealBrushTests.cs

```diff
@@ -94,8 +94,7 @@ public void RevealAlwaysUseFallback()
                 return;
             }
 
-            using (IDisposable page1 = new TestSetupHelper("Reveal Tests"),
-                               page2 = new TestSetupHelper("navigateToRevealFallback"))
+            using (var setup = new TestSetupHelper(new string[] { "Reveal Tests", "navigateToRevealFallback" }))
```

Comment:
can simplify to new[] {"", ""} -- compiler can infer the "string" part of this. Just a little simpler. :)

Reply:
Optionally, the ctor could take the array as "params string[] testNames", so you could just do "new TestSetupHelper("Reveal Tests", "navigateToRevealFallback")".

Reply:
Params has to be the last parameter, we have other parameters with default values that need to come last too. I tried adding another overloaded constructor takes only params string[], the compiler get confused, new TestSetupHelper("A", "B") goes to TestSetupHelper(string testName, string languageOverride = "", bool attemptRestartOnDispose = true) instead of the overloaded params version.

I guess I'll just use new[] {"", ""} for now.

Reply:
That makes sense.